bootstrap/config/kfctl_k8s_istio.yaml

```diff
@@ -1,150 +1,247 @@
-    istio-crds:
-    - name: namespace
-      value: istio-system
-    istio-install:
-    - name: namespace
-      value: istio-system
-    jupyter:
-    - name: overlay
-      value: application
-    jupyter-web-app:
-    - name: overlay
-      value: istio
-    - name: overlay
-      value: application
-    katib-ui: # Issue: https://github.com/kubeflow/manifests/issues/151
-    - name: overlay
-      value: istio
-    metadata:
-    - name: overlay
-      value: istio
-    minio:
-    - name: minioPd
-      value: dls-kf-storage-artifact-store
-    - name: minioPvName
-      value: minio-pv
-    - name: minioPvcName
-      value: minio-pv-claim
-    mysql:
-    - name: mysqlPd
-      value: dls-kf-storage-metadata-store
-    - name: mysqlPvName
-      value: mysql-pv
-    - name: mysqlPvcName
-      value: mysql-pv-claim
-    notebook-controller:
-    - name: overlay
-      value: istio
-    - name: overlay
-      value: application
-    pipelines-ui:
-    - name: overlay
-      value: istio
-    profiles:
-    - name: overlay
-      value: istio
-    - initRequired: true
-      name: admin
-      value: johnDoe@acme.com
-    spartakus:
-    - initRequired: true
-      name: usageId
-      value: <randomly-generated-id>
-    - initRequired: true
-      name: reportUsage
-      value: "true"
-    tensorboard:
-    - name: overlay
-      value: istio
-    tf-job-operator:
-    - name: overlay
-      value: istio
-    seldon-core-operator:
-    - name: overlay
-      value: application
-  components:
-  # Istio install. If not needed, comment out istio-crds and istio-install.
-  - istio-crds
-  - istio-install
-  # This component is the istio resources for Kubeflow (e.g. gateway), not about installing istio.
-  - istio
-  - application-crds
-  - application
-  - metacontroller
-  - argo
-  - centraldashboard
-  - bootstrap
-  - webhook
-  - jupyter-web-app
-  - katib-db
-  - katib-manager
-  - katib-controller
-  - katib-ui
-  - metadata
-  - metrics-collector
-  - suggestion
-  - notebook-controller
-  - pytorch-job-crds
-  - pytorch-operator
-  - spartakus
-  - tensorboard
-  - tf-job-operator
-  - api-service
-  - minio
-  - mysql
-  - persistent-agent
-  - pipelines-runner
-  - pipelines-ui
-  - pipelines-viewer
-  - scheduledworkflow
-  - profiles
-  - seldon-core-operator
-  enableApplications: true
-  packageManager: kustomize
-  packages:
-  - argo
-  - common
-  - examples
-  - admission-webhook
-  - istio
-  - jupyter
-  - katib-v1alpha2
-  - metacontroller
-  - metadata
-  - modeldb
-  - mpi-job
-  - pytorch-job
-  - seldon
-  - tensorboard
-  - tf-serving
-  - tf-training
-  - pipeline
-  - profiles
-  - application
-  useIstio: true
   repos:
   - name: manifests
     root: manifests-master
     uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
   - name: kubeflow
     root: kubeflow-master
     uri: https://github.com/kubeflow/kubeflow/archive/master.tar.gz
+  appdir: SET_APP_DIR
```

Comment:
I don't think we need appdir anymore. Can you try removing it and seeing if it works?

Reply:
This works without appdir.